src/main/java/com/webapp/TextBook/sharedFiles/StatusCode.java

```diff
@@ -0,0 +1,37 @@
+package com.webapp.TextBook.sharedFiles;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public enum StatusCode {
+    BookCopyUndefined(""),
```

Comment:
@Infraction71 
I'm tasking you with adding user-friendly error messages to these enums (ie: for DatabaseError "An internal error occurred while executing; please contact IT Support")
IF the error is not business related (ie: for BookCopyUndefined "The requested strike barcode does not match a known book in our records.") then use the generic error I gave above (or word it better if you don't like it)